src/content_scripts/common/utils.js

```diff
@@ -589,8 +589,10 @@ function getAnnotations(mappings) {
 }
 
 function constructSearchURL(se, word) {
-    if (se.indexOf("{0}") > 0) {
+    if (se.indexOf("{0}") !== -1) {
```

Comment:
Looks good, but is it necessary to support the case that the placeholders appear at the beginning(ie. equals zero)?

Reply:
honestly, probably not...the other function did, but can't think of a case where that would be useful
```suggestion
    if (se.indexOf("{0}") > 0) {
```